aospy/test/test_calc_basic.py

```diff
@@ -37,6 +39,7 @@ def test_annual_mean(self):
         calc.compute()
         assert isfile(calc.path_out['av'])
         assert isfile(calc.path_tar_out)
+        _test_output_attrs(calc, 'av')
```

Comment:
I would like to add this test to all of the cases.  In fact, we should bundle this along with the two `assert` statements that are in all of these.  E.g. define at the module level

```python
def _test_files_and_attrs(calc, dtype_out):
    assert isfile(calc.path_out[dtype_out])
    assert isfile(calc.path_tar_out)
    _test_output_attrs(calc, 'av')

And then replace the two assert statements (and `_test_output_attrs` where you added it) in each of these tests with this single function call.